### PR TITLE
Adding analyzer button sensitivity

### PR DESCRIPTION
**Pull request by egbicker**

### CODE REVIEW

**Comment by tparchambault** [comment]
Sensitivity appears to be correct, however the application is not automatically transitioning to the Analysis tool pane.

Tested over FC34 via an rpm install.

**Comment by tparchambault** [comment]
All good over FC34 via rpm install as the root user.

File: fapolicy_analyzer/ui/profiler_page.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ def __init__(self, fapd_manager):
         UIConnectedWidget.__init__(self, get_system_feature())
         self.__events__ = [
             "analyze_button_pushed",
+            "refresh_toolbar",
```
Comment (tparchambault) [inline fapolicy_analyzer/ui/profiler_page.py:35]: Maybe at some point, "refresh_toolbar" should be converted to something on the order of an event, e.g. "analysis complete" which might map into a redux state var if needed.

**Review by tparchambault — APPROVED** [review]

**Review by jw3 — APPROVED** [review]
Functions as expected

**Comment by jw3** [comment]
Found #632 while testing this.

**Comment by jw3** [comment]
@dorschs57 are you good with this?

**Review by dorschs57 — APPROVED** [review]
I'm OK with this. I'm not a big fan of needing to reach back to the `MainWindow` to force a toolbar refresh, but I can't think of a better way to do it right now. Ultimately I think the toolbar and menu items need to be wired into redux like we talked about before, but that's a bigger issue.

**Comment by egbicker** [comment]
> I'm OK with this. I'm not a big fan of needing to reach back to the `MainWindow` to force a toolbar refresh, but I can't think of a better way to do it right now. Ultimately I think the toolbar and menu items need to be wired into redux like we talked about before, but that's a bigger issue.

Completely agree

**Pull request merged**